bookbotproject/conversationtree/models.py

```diff
@@ -56,12 +56,13 @@ class Book(models.Model):
     # Every book has unique isbn
     isbn = models.CharField(max_length=100, blank = True)
     # We may know the title of the book.
-    title = models.CharField(max_length=100)
+    title = models.CharField(unique=True, max_length=100)
```

Comment:
Titles can be the same but authors may be different, please do not make this field unique.